src/node_snapshotable.cc

```diff
@@ -584,7 +584,7 @@ size_t SnapshotSerializer::Write(const SnapshotMetadata& data) {
 // [    ...       ]  env_info
 // [    ...       ]  code_cache
 
-std::vector<char> SnapshotData::ToBlob() const {
+void SnapshotData::ToBlob(std::vector<char>* out) const {
```

Comment:
Why do we need this change? In `src/node_sea.cc`, we can do `snapshot_blob = snapshot.ToBlob();` and keep using the existing API instead of creating another one, right?

Reply:
I did it to avoid the copy (which would be there if we just do the assign), though I think we could probably do a move assignment...